src/tree_sortable.rs

```diff
@@ -0,0 +1,144 @@
+// This file was generated by gir (17af302) from gir-files (11e0e6d)
+// DO NOT EDIT
+
+use SortType;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use auto::traits::TreeSortableExt as Auto;
+use std::thread;
+use std::cell::RefCell;
+use std::mem::transmute;
+use std::process;
+use std::cmp::Ordering;
+
+use glib_ffi::gpointer;
+use {TreeSortable, TreeIter};
+use ffi::{GtkTreeIter, GtkTreeSortable};
+
+pub enum SortColumn {
+    Default,
+    Unsorted,
+    Column(u32),
+}
+
+impl ToGlib for SortColumn {
+    type GlibType = i32;
+
+    #[inline]
+    fn to_glib(&self) -> i32 {
+        match *self {
+            SortColumn::Column(x) => x as i32,
+            SortColumn::Default => ffi::GTK_TREE_SORTABLE_DEFAULT_SORT_COLUMN_ID,
+            SortColumn::Unsorted => ffi::GTK_TREE_SORTABLE_UNSORTED_SORT_COLUMN_ID,
+        }
+    }
+}
+
+impl FromGlib<i32> for SortColumn {
+    #[inline]
+    fn from_glib(val: i32) -> SortColumn {
+        if val == ffi::GTK_TREE_SORTABLE_DEFAULT_SORT_COLUMN_ID {
+            SortColumn::Default
+        } else if val == ffi::GTK_TREE_SORTABLE_UNSORTED_SORT_COLUMN_ID {
+            SortColumn::Unsorted
+        } else {
+            SortColumn::Column(val as u32)
+        }
```

Comment:
How about using a match instead of an `if else if` match?

```
match val {
    ffi::GTK_TREE_SORTABLE_DEFAULT_SORT_COLUMN_ID => SortColumn::Default,
    ffi::GTK_TREE_SORTABLE_UNSORTED_SORT_COLUMN_ID => SortColumn::Unsorted,
    _ => SortColumn::Column(val as u32)
}